server/services/userAnswerService.js

```diff
@@ -0,0 +1,44 @@
+const UserAnswer = require('../models/userAnswer');
+
+const parseUserAnswers = (data) => {
+    return Object.entries(data).map(([questionId, response]) => ({
+        question: questionId,
+        response
+    }));
+}
+const insertUserAnswers = async (userAnswers) => {
+    return await UserAnswer.insertMany(userAnswers);
+}
+
+const getUsersWithAnswers = async(user, userAnswer) => {
```

Comment:
it seems that function is exported but I can't find where it's used

Reply:
You're right. It is a part of unfinished functionality. It will be used for simplifying the logic in http handler.